### PR TITLE
[functions] improve docs markup

### PR DESCRIPTION
**Pull request by Kikobeats**

### CODE REVIEW

**Review by Schniz — DISMISSED** [review]
<picture data-single-emoji=":just-put-it:" title=":just-put-it:">:just-put-it:</picture>

**Review by Schniz — APPROVED** [review]

File: packages/functions/docs/modules/index.md [hunk 1]
```diff
@@ -40,16 +43,36 @@ Returns the location information for the incoming request.
...
+ "region": "dev1",
+ "latitude": "40.7128",
+ "longitude": "-74.0060"
+}
+```
+
 #### Defined in
 
-[packages/functions/src/headers.ts:128](https://github.com/vercel/vercel/blob/main/packages/functions/src/headers.ts#L128)
+[packages/functions/src/headers.ts:158](https://github.com/vercel/vercel/blob/main/packages/functions/src/headers.ts#L158)
```
Comment (EndangeredMassa) [inline packages/functions/docs/modules/index.md:62]: **suggestion (soft-blocking):** use canonical link

This link goes to `main`, which doesn't have this line number.

File: packages/functions/docs/modules/index.md [hunk 2]
```diff
@@ -86,7 +109,7 @@ Returns the location information for the incoming request.
 
 #### Defined in
 
-[packages/functions/src/get-env.ts:1](https://github.com/vercel/vercel/blob/main/packages/functions/src/get-env.ts#L1)
+[packages/functions/src/get-env.ts:6](https://github.com/vercel/vercel/blob/main/packages/functions/src/get-env.ts#L6)
```
Comment (EndangeredMassa) [inline packages/functions/docs/modules/index.md:112]: **suggestion (soft-blocking):** use canonical link

File: packages/functions/docs/modules/index.md [hunk 3]
```diff
@@ -110,9 +140,11 @@ IP_HEADER_NAME
 
 `string` \| `undefined`
 
+The IP address of the request.
+
 #### Defined in
 
-[packages/functions/src/headers.ts:99](https://github.com/vercel/vercel/blob/main/packages/functions/src/headers.ts#L99)
+[packages/functions/src/headers.ts:111](https://github.com/vercel/vercel/blob/main/packages/functions/src/headers.ts#L111)
```
Comment (EndangeredMassa) [inline packages/functions/docs/modules/index.md:147]: **suggestion (soft-blocking):** use canonical link

File: packages/functions/docs/modules/oidc.md [hunk 4]
```diff
@@ -99,4 +95,4 @@ A promise that resolves to the OIDC token.
 
 #### Defined in
 
-[packages/functions/src/oidc/get-vercel-oidc-token.ts:23](https://github.com/vercel/vercel/blob/main/packages/functions/src/oidc/get-vercel-oidc-token.ts#L23)
+[packages/functions/src/oidc/get-vercel-oidc-token.ts:24](https://github.com/vercel/vercel/blob/main/packages/functions/src/oidc/get-vercel-oidc-token.ts#L24)
```
Comment (EndangeredMassa) [inline packages/functions/docs/modules/oidc.md:98]: **suggestion (soft-blocking):** use canonical link

This link goes to `main`, which doesn't have this line number.

**Review by EndangeredMassa — APPROVED** [review]

**Pull request merged**